### ISSUE TITLE
scoring-sssq

### ISSUE THREAD

**Issue by jessb0t** [NDCLab/instruments#71]

**Comment by jessb0t** [comment]
@georgebuzzell Please take a look at the scoring in the issue description to confirm before we formalize it in the script.  Thank you! :pray:

**Comment by georgebuzzell** [comment]
@jessb0t and @anaNDClab looks good with one caveat/question:

I notice the delta scores are percentage change scores (which I think are also called relative delta). percentage change is great, but not the standard typically. Apologies if I misremember discussing percentage change vs simple difference (delta), if we did and we had settled on percentage change please just remind me! Or, if the sssq typically reccomends delta, please let me know! otherwise, i would just say to please change the default deltas to be simple subtraction scores. thanks!

**Comment by jessb0t** [comment]
@anaNDClab Could you take a look and see if the SSSQ recommends percentage change or simple subtraction delta?  Thanks!

**Comment by jessb0t** [comment]
@F-said I removed three of the subscores above as I realized they weren't necessary.

**Comment by anaNDClab** [comment]
> @anaNDClab Could you take a look and see if the SSSQ recommends percentage change or simple subtraction delta? Thanks!

@georgebuzzell  and @jessb0t In the Helton and Näsall (2015) paper, they use simple subtraction delta, and I do not see any recommendations to use percentage change.

![sssq](https://user-images.githubusercontent.com/86448434/151450590-d3c9e024-debd-474c-93ca-f25de62d376b.png)

Do I go ahead and change the scoring to reflect the simple subtraction?

**Comment by jessb0t** [comment]
@anaNDClab Yes, please!  I will update on the repo readme file if you update in this issue! :pray:

**Comment by anaNDClab** [comment]
> @anaNDClab Yes, please! I will update on the repo readme file if you update in this issue! 🙏

Done!

**Comment by jessb0t** [comment]
@F-said You unassigned yourself.  Are you still going to code this instrument into the script for the v0.02 release?
